gen/calc.py

```diff
@@ -1082,6 +1083,7 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'check_search_path': CHECK_SEARCH_PATH,
         'mesos_master_work_dir': '/var/lib/dcos/mesos/master',
         'mesos_agent_work_dir': '/var/lib/mesos/slave',
+        'dcos_diagnostics_bundles_dir': '/var/lib/dcos/dcos-diagnostics/diag-bundles',
```

Comment:
@klueska Who is responsible for creating this directory? If I change it to `/var/log` where can I configure permissions and ownership?

Reply:
I like having the mkdir + chown in the unit. Could you put that back, please?

Reply:
If we assign this param's value to an env var here, the unit can reference that env var and create the directory: https://github.com/dcos/dcos/blob/225b085d6097a29784ee244fd92b1d360d072ab4/gen/dcos-config.yaml#L673-L675

Reply:
Fixed